CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,132 @@
+
+All members of the Istio community must abide by the following Code of Conduct,
+adapted from the [Contributor Covenant][homepage]. Only by respecting each
+other can we develop a productive, collaborative community.
+
+## Our Pledge
+
+As members, contributors, and leaders of the Istio community, we pledge to make
+participation in our community a harassment-free experience for everyone,
+regardless of age, body size, visible or invisible disability, ethnicity, sex
+characteristics, gender identity and expression, level of experience,
+education, socio-economic status, nationality, personal appearance, race,
+religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders (including maintainers, Working Group leads, the Technical
+Oversight Committee and the Steering Committee) are responsible for clarifying
+and enforcing our standards of acceptable behavior and will take appropriate
+and fair corrective action in response to any behavior that they deem
+inappropriate, threatening, offensive, or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the Istio Steering Committee at steering@discuss.istio.io.
```

Comment:
or any individual who serves on the steering committee.

Reply:
I think adding this is appropriate - we had it before.  It is an option that may bring comfort to folks who may prefer to select a person one trusts on the steering committee (vs sending to all who subscribed to the mailing list).

Reply:
@ssuchter to update this to CoC specific mailing list; at the same time we can include "or individual members of Steering" IMO.

Reply:
Done.